tests/integration/test_fim/test_files/test_max_files_per_second/test_max_files_per_second.py

```diff
@@ -1,11 +1,79 @@
-# Copyright (C) 2015-2021, Wazuh Inc.
-# Created by Wazuh, Inc. <info@wazuh.com>.
-# This program is free software; you can redistribute it and/or modify it under the terms of GPLv2
+'''
+copyright:
+    Copyright (C) 2015-2021, Wazuh Inc.
+
+    Created by Wazuh, Inc. <info@wazuh.com>.
+
+    This program is free software; you can redistribute it and/or modify it under the terms of GPLv2
+
+type:
+    integration
+
+brief:
+    Check that FIM sleeps for one second when the option max_files_per_second is enabled
+
+tier:
+    1
+
+modules:
+    - syscheck
+
+components:
+    - manager
+
+path:
+    tests/integration/test_fim/test_files/test_max_files_per_second/test_max_files_per_second.py
+
+daemons:
+    - wazuh-modulesd
+    - wazuh-db
+
+os_platform:
+    - linux
+
+os_version:
+    - Amazon Linux 1
+    - Amazon Linux 2
+    - Arch Linux
+    - CentOS 6
+    - CentOS 7
+    - CentOS 8
+    - Debian Buster
+    - Debian Stretch
+    - Debian Jessie
+    - Debian Wheezy
+    - Red Hat 6
+    - Red Hat 7
+    - Red Hat 8
+    - Ubuntu Bionic
+    - Ubuntu Trusty
+    - Ubuntu Xenial
```

Comment:
Add the Windows platforms with all os_versions versions, or remove all os_versions of Linux and Windows, and in the future, we will add it.